homeassistant/components/jellyfin/sensor.py

```diff
@@ -35,8 +35,6 @@ def _count_now_playing(data: JellyfinDataT) -> int:
     "sessions": JellyfinSensorEntityDescription(
         key="watching",
         translation_key="watching",
-        name=None,
-        native_unit_of_measurement="Watching",
```

Comment:
Wouldn't the unit of measurements not be `clients`?

Reply:
The label is now "Active clients", do you think the `native_unit_of_measurement` is needed at all?

Reply:
I think so yes